### PR TITLE
Create cpp17.cpp for problem 1001

### PR DESCRIPTION
**Pull request by tasfik007**
## Issue Number :
#8

### CODE REVIEW

File: 1001/cpp17.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include<bits/stdc++.h>
```
Comment (jan876) [inline 1001/cpp17.cpp]: minor spacing: `#include <bits/stdc++.h>`

File: 1001/cpp17.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+using namespace std;
+
+/**
+ * Returns  the number of problems stored in each computer.
+ * @param n denotes the total number of problems.
+ */
+vector<int> problemsStored(int n){
```
Comment (jan876) [inline 1001/cpp17.cpp]: C++ styling: use a space before `{`

File: 1001/cpp17.cpp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    // Implement this method
+
+}
+
+/**
+ * Takes care of the problem input and output.
+ */
+int main(){
```
Comment (jan876) [inline 1001/cpp17.cpp]: Same comment as above.

File: 1001/cpp17.cpp [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    int T;
```
Comment (jan876) [inline 1001/cpp17.cpp]: Use qualified names, `cases, caseno`.

File: 1001/cpp17.cpp [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    scanf("%d", &T);
+    while(T--){
```
Comment (jan876) [inline 1001/cpp17.cpp]: use a for loop, with `caseno`.

Also, the c++ styling standard wise, use space after while, if, for, etc.

File: 1001/cpp17.cpp [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        int n;
+        scanf("%d", &n);
+        
+        vector<int> ans = problemsStored(n);
+        printf("%d %d\n",ans[0], ans[1]);
```
Comment (jan876) [inline 1001/cpp17.cpp]: Use space after comma.

File: 1001/cpp17.cpp [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    }
+    return 0;
+}
```
Comment (jan876) [inline 1001/cpp17.cpp]: Add a newline after the last line.

**Review by jan876 — CHANGES_REQUESTED** [review]
Thanks a lot. Overall looks good, few minor comments.

**Comment by tasfik007** [comment]
Sure, I will make the changes and thanks for the quick feedback.

File: 1001/cpp17.cpp (hunk 7 above)
Comment (tasfik007) [reply 1001/cpp17.cpp]: newline added at the end.

**Comment by jan876** [comment]
Awesome!

**Comment by jan876** [comment]
Merged.

**Pull request merged**